osquery/tables/system/darwin/battery.mm

```diff
@@ -0,0 +1,192 @@
+/*
+ *
+ *  Copyright (c) 2018-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <CoreFoundation/CoreFoundation.h>
+#include <Foundation/Foundation.h>
+
+#include <osquery/tables.h>
+
```

Comment:
nit: if possible, can you bring this include below the IOkit includes to keep our “broadest to narrowest” format?